<launch>
    <arg name="robot_name" default="echo"/>
    <group ns="$(arg robot_name)">
        <param name="tf_prefix" value="$(arg robot_name)"/>

        <node name="abridge" pkg="abridge" type="abridge">
            <param name="devicePath" value="/dev/arduino"/>
            <param name="baudRate" value="115200"/>
            <param name="timerPeriod" value="0.1"/>
        </node>
    </group>
</launch>
